packages/devtools_app/lib/src/bloc/models/bloc_object.dart

```diff
@@ -0,0 +1,4 @@
+class BlocObject {
```

Comment:
If this object is being used within `Equatable` props it should also extend `Equatable` or otherwise override `==` and `hashCode` to make it comparable.